plugins/org.yakindu.sct.doc.user/src/user-guide/simulating_statecharts.textile

```diff
@@ -278,108 +296,129 @@ h4(#advsim_continuing_the_simulation). Continuing the simulation
 * the breakpoint is enabled.
 
 In order to attach a condition to a breakpoint, proceed as follows:
-* In the _breakpoints_ view, select the breakpoint in question.
+* In the breakpoints view, select the breakpoint in question.
 * Check the _Conditional_ checkbox, see "figure &quot;Breakpoints view&quot;":#advsim_fig_breakpoints_view in the lower right area. The associated text field becomes writable.
+* Enter the condition into the text field. Like in the statechart editor, a content assist is available when pressing @[Ctrl+Space]@. The expression you entered into the text field is validated automatically. In the example shown in "figure &quot;Breakpoints view&quot;":#advsim_fig_breakpoints_view, the transition suspends the simulation only if the variable _brightness_ has a value of 4.
+
 
-Enter the condition into the text field. Like in the statechart editor, a content assist is available when pressing @[Ctrl+Space]@. The expression is validated automatically. In the example shown in "figure &quot;Breakpoints view&quot;":#advsim_fig_breakpoints_view the transition suspends the simulation only if the variable _brightness_ has a value of 4.
 
 h3(#advsim_debugging_a_statechart). Debugging a statechart
 
 h4(#advsim_changing_variable_values). Changing variable values
 
-In the suspended status of a statechart simulation you can change variable values using the _simulation_ view. When continuing execution – see section "Continuing the simulation":#advsim_continuing_the_simulation – you can observe how your state machine behaves with those modified values.
+In the suspended status of a statechart simulation, you can change variable values using the simulation view. When continuing execution – see section "Continuing the simulation":#advsim_continuing_the_simulation – you can observe how your state machine behaves with those modified values.
+
 
 h4(#advsim_raising_multiple_events_simultaneously). Raising multiple events simultaneously
 
-If you click on an event's name in the _simulation_ view to raise that event in normal simulation, i.e., while execution isn't suspended, the state machine immediately processes that event and takes the corresponding transition, if any.
+If you click on an event's name in the simulation view to raise that event in normal simulation, i.e., while execution isn't suspended, the state machine immediately processes that event and takes the corresponding transition, if any.
+
+However, while the simulation is suspended, you can raise multiple events, without instant execution. Once execution resumes, the state machine's behavior depends on its "execution scheme":../user-guide/code_generation_intro.html#codegen_execution_schemes:
+* In the _cycle-based_ execution scheme, both events are handled at the same time, or, to be more exact, in the same run-to-completion step (RTC).
+* In the _event-driven_ execution scheme, each of the two events will trigger a run-to-completion step (RTC) of its own.
```

Comment:
This is currently not implemented because in events are not queued. Only one event is processed.